HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -1090,6 +1090,172 @@ def customizeHLTforAlpakaEcalLocalReco(process):
+
+    # run the HCAL local reconstruction, potentially offloading the MHI step to the device
+    process.HLTDoLocalHcalSequence = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoA +
+      process.hltHbheRecoSoA +
+      process.hltHbhereco +
+      process.hltHfprereco +
+      process.hltHfreco +
+      process.hltHoreco )
+
+    # run the HCAL local reconstruction on the host
+    process.HLTDoLocalHcalSequenceSerialSync = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoASerialSync +
+      process.hltHbheRecoSoASerialSync +
+      process.hltHbherecoSerialSync +
+      process.hltHfprereco +
+      process.hltHfreco +
+      process.hltHoreco )
+
+    # run the HBHE local reconstruction, potentially offloading the MHI step to the device
+    process.HLTStoppedHSCPLocalHcalReco = cms.Sequence(
+      process.hltHcalDigis +
+      process.hltHcalDigisSoA +
+      process.hltHbheRecoSoA +
+      process.hltHbhereco )
+
+    # PFJet reconstruction running on the host
+    process.HLTPFHcalClusteringSerialSync = cms.Sequence(
+      process.hltHbheRecHitSoASerialSync +
+      process.hltParticleFlowRecHitHBHESoASerialSync +
+      process.hltParticleFlowRecHitHBHESerialSync +
+      process.hltParticleFlowClusterHBHESoASerialSync +
+      process.hltParticleFlowClusterHBHESerialSync +
+      process.hltParticleFlowClusterHCALSerialSync )
+
+    paths_AlCa_PFJet40_CPUOnly_v = [ path for path in process.paths if path.startswith('AlCa_PFJet40_CPUOnly_v') ]
+    if paths_AlCa_PFJet40_CPUOnly_v:
+      pathname = paths_AlCa_PFJet40_CPUOnly_v[0]
+      setattr(process, pathname, cms.Path(
+        process.HLTBeginSequence +
+        process.hltL1sZeroBias +
+        process.hltPreAlCaPFJet40CPUOnly +
+        process.HLTDoLocalHcalSequenceSerialSync +
+        process.HLTAK4CaloJetsSequenceSerialSync +
+        process.hltSingleCaloJet10SerialSync +
+        process.HLTAK4PFJetsSequenceSerialSync +
+        process.hltPFJetsCorrectedMatchedToCaloJets10SerialSync +
+        process.hltSinglePFJet40SerialSync +
+        process.HLTEndSequence )
+      )
+
+    # compare the HCAL local reconstruction running on the device and on the host
+    paths_DQM_HcalReconstruction_v = [ path for path in process.paths if path.startswith('DQM_HcalReconstruction_v') ]
+    if paths_DQM_HcalReconstruction_v:
+      pathname = paths_DQM_HcalReconstruction_v[0]
+      setattr(process, pathname, cms.Path(
+        process.HLTBeginSequence +
+        process.hltL1sDQMHcalReconstruction +
+        process.hltPreDQMHcalReconstruction +
+        process.hltBackend +
+        process.hltStatusOnGPUFilter +
+        process.HLTDoLocalHcalSequence +
+        process.HLTDoLocalHcalSequenceSerialSync +
+        process.HLTPFHcalClustering +
+        process.HLTPFHcalClusteringSerialSync +
+        process.HLTEndSequence )
+      )
+
```

Comment:
```suggestion
+    # modify EventContent of *DQMGPUvsCPU streams
+    for hltOutModMatch in ['hltOutputDQMGPUvsCPU', 'hltOutputHIDQMGPUvsCPU']:
+        if hasattr(process, hltOutModMatch):
+            outMod = getattr(process, hltOutModMatch)
+            outCmds_new = [foo for foo in outMod.outputCommands if 'Hcal' not in foo]
+            outCmds_new += [
+                'keep *_hltHcalDigis_*_*',
+                'keep *_hltHcalDigisSoASerialSync_*_*',
+                'keep *_hltHbhereco_*_*',
+                'keep *_hltHbherecoSerialSync_*_*',
+            ]
+            outMod.outputCommands = outCmds_new[:]
```

Reply:
@kakwok there are two issues here;
   * looking at the current EventContent you need to match the `Hbhe` and not `Hcal` string.
   * it's not clear why you need to save the hcal digis (not consumed by the DQM client).
   
I would propose to use: 

```python
 # modify EventContent of *DQMGPUvsCPU streams
    for hltOutModMatch in ['hltOutputDQMGPUvsCPU', 'hltOutputHIDQMGPUvsCPU']:
        if hasattr(process, hltOutModMatch):
            outMod = getattr(process, hltOutModMatch)
            outCmds_new = [foo for foo in outMod.outputCommands if 'Hbhe' not in foo]
            outCmds_new += [
                'keep *_hltHbhereco_*_*',
                'keep *_hltHbherecoSerialSync_*_*',
            ]
            outMod.outputCommands = outCmds_new[:]   
```

Reply:
same here.